examples/web_client/index.js

```diff
@@ -121,6 +122,26 @@ async function main () {
       }
     }, false)
 
+    document.addEventListener('mousedown', (e) => {
+      const BlockDistance = (bot.gamemode == 1) ? 8 : 5
```

Comment:
where do these constants come from?

Reply:
Just checked the wiki https://minecraft.gamepedia.com/Breaking#Basics_of_breaking it should be 5 for creative and 4.5 for survival.

Reply:
Oh someone on the server told me 8 for creative and 5 for survival

Reply:
Yeah, that was me, but you should have checked :)